packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -2134,14 +2135,12 @@ describe("Teal Opcodes", function () {
     });
 
     it("should push LatestTimestamp to stack", function () {
+      interpreter.runtime.setBlockGlobalData(500, 100);
       const op = new Global(['LatestTimestamp'], 1, interpreter);
       op.execute(stack);
 
       const top = stack.pop();
-      const ts = Math.round((new Date()).getTime() / 1000);
-
-      const diff = Math.abs(ts - Number(top)); // for accuracy
-      assert.isBelow(diff, 10);
+      assert.equal(BigInt('100'), top);
```

Comment:
we can use `100n` instead of `BigInt('100')`

Reply:
ok